CONSTITUTION.md

```diff
@@ -214,6 +215,10 @@ Dissolution of the Society shall take place only if such a decision is reached b
 
 ## 10. CONSTITUTIONAL AMBIGUITY
 
-### 10.1. Executive Committee Ruling
+### 10.1. ActivateUTS Club Affiliation Agreement
+
+In order to fulfil the object of remaining affiliated with ActivateUTS, the ActivateUTS Club Affiliation Agreement shall supersede any inconsistencies with this constitution to the extent of the inconsistency.
+
+### 10.2. Executive Committee Ruling
 
 In the case of doubt or conflict of opinion in interpretation of this constitution or in any circumstance not covered by this constitution, the Executive Committee's ruling shall be binding upon members.
```

Comment:
Insert after the second instance of "this constitution" and before "the Executive Committee's ruling"

"or the ActivateUTS Club Affiliation Agreement"

Reply:
Agreed, this makes things explicit.